tools/bin/commands/util/operator_funcs

```diff
@@ -100,7 +100,9 @@ download_operator_binary() {
             | grep browser_download_url \
             | grep ${OPERATOR} \
             | grep ${pattern} \
-            | cut -d '"' -f 4)
+            | grep -v 'download/2' \
+            | cut -d '"' -f 4 \
+            | head -1)
```

Comment:
Would it make a more sense to use `jq` here? For example based on this:

```shell
curl -s https://api.github.com/repos/syndesisio/syndesis/releases\?per_page\=4 \
| jq -r '.[]| select(.tag_name == "1.12.0-20210325")| .assets[] | select(.name |startswith("syndesis-operator-linux-amd64"))| .browser_download_url'
```

Reply:
Using jq now.